src/Plugin/ultimate_cron/Logger/DatabaseLogger.php

```diff
@@ -202,7 +202,7 @@ public function settingsForm(&$form, &$form_state, $job = NULL) {
   }
 
   /**
-   * Submit handler.
+   * {@inheritdoc}
    */
   public function settingsFormSubmit(&$form, &$form_state, $job = NULL) {
```

Comment:
I'm not sure if those settings methods are still called, very likely not, because this is not D8 code. We can possibly remove it. But, separately. Might not want to add it to the interface, though.